verification/curator-service/ui/src/components/Users.test.tsx

```diff
@@ -1,6 +1,7 @@
-import Users from './Users';
+import { fireEvent, render, within } from '@testing-library/react';
```

Comment:
Out of curiosity, do you manually reorder these imports or does it happen on save? I don't have any automatic reordering of imports configured.

Reply:
I do have the sort-imports vs code extension installed yes and recommend it.
https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports